docs/ci/demo

```diff
@@ -231,9 +231,11 @@ popper rm mypipetwo
 init_test
 popper search quiho
 test -d ./.cache
+popper search --list-repos
 popper search quiho --skip-update
 popper search linux --skip-update
-
+popper search linux --rm popperized
```

Comment:
maybe `cat` the config file so that we make sure that `popperized` is not there after we removed it?